### PR TITLE
Placed RPM release in a distinct directory and generated a repository

### PR DESCRIPTION
**Pull request by stevekuznetsov**
When creating a RPM release, it is useful to keep the generated RPM
artifacts separate from the rest of the artifacts in the release out-
put directory. Furthermore, if the user has `createrepo` installed,
we can go ahead anad create a repository and .repo file for them for
the local release so they may install the generated RPMs with `yum`.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
@smarterclayton is this what you were looking for

File: hack/build-rpm-release.sh [hunk 1]
```diff
@@ -53,5 +53,21 @@ make clean
 # migrate the tito artifacts to the Origin directory
 mkdir -p "${OS_OUTPUT}"
 cp -r "${tito_output_directory}"/* "${OS_OUTPUT}"
-mkdir -p "${OS_LOCAL_RELEASEPATH}"
-cp "${tito_tmp_dir}"/x86_64/*.rpm "${OS_LOCAL_RELEASEPATH}"
\ No newline at end of file
+mkdir -p "${OS_LOCAL_RELEASEPATH}/rpms"
+cp "${tito_tmp_dir}"/x86_64/*.rpm "${OS_LOCAL_RELEASEPATH}/rpms"
```
Comment (smarterclayton) [inline hack/build-rpm-release.sh]: I think you want to preserve the directory structure

Comment (smarterclayton) [inline hack/build-rpm-release.sh]: The arch that is

Comment (smarterclayton) [inline hack/build-rpm-release.sh]: Don't I need repodata/repomod.xml?

Comment (stevekuznetsov) [inline hack/build-rpm-release.sh]: RPMs are placed flat in the output from `tito`, and `createrepo` below is what is actually building the repo and metadata

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**